commands/utils/docker-endpoint.ts

```diff
@@ -0,0 +1,43 @@
+import * as Docker from 'dockerode';
+
+
+class DockerClient {
+
+    private endPoint:Docker;
+
+    constructor() {
+        this.endPoint = new Docker({ socketPath: '/var/run/docker.sock' });
```

Comment:
Would it make sense to omit the `socketPath`, and instead allow the `DOCKER_HOST` env var to determine what this extension connects to? That way, a dev could use these VS Code commands even if they had configured their workstation to manage a remote host (e.g. via docker-machine).